app/_data/distributions.yml

```diff
@@ -42,8 +42,8 @@
   name: Ubuntu
   type: package
 
-- id: osx
-  name: OS X
+- id: macos
```

Comment:
This breaks navigation from https://konghq.com/install/ to https://docs.konghq.com/install/osx/ - instead of dealing with updating that link, and adding the necessary redirect, I'm going to revert this one change. The result will be that the URL will still say `osx` but on all the pages of docs.konghq.com it'll say `macOS` (thanks to your helpful update!)

Reply:
I was writing something in response to that PR (thank you @arshadkazmi42! This was actually on our TODO list for way too long).

Updating the URL to `/macos` from `/osx` is something that should be within the scope of this PR. Instead of reverting the change here, we should rename the `osx.md` file to `macos.md` (fixing the broken link), but also update `osx.md` to redirect to `/macos`, which should be straightforward.

Reply:
I did that update, thanks for pointing me in the right direction @thibaultcha